include/deal.II/base/vectorization.h

```diff
@@ -521,6 +521,23 @@ inline DEAL_II_ALWAYS_INLINE VectorizedArray<Number, width>
 
 
 
+/**
+ * Create a vectorized array that sets all entries in the array to the given
+ * scalar.
+ *
+ *  @relatesalso VectorizedArray
+ */
+template <typename VectorizedArrayType>
+inline DEAL_II_ALWAYS_INLINE VectorizedArrayType
+                             make_vectorized_array(const typename VectorizedArrayType::value_type &u)
```

Comment:
This requires you to write "vectorized" and "array" twice, once in the function name and once in the template argument (in the form of the class name) you need to specify explicitly here. I think you might want to use this instead:
```
template <typename Number, int width>
VectorizedArray<Number,width>
make_vectorized_array (const Number &u) {...}
```
instead.

Reply:
I see your point @bangerth. You are thinking about such a case: you have given the value type `Number` and the vector length `width`:
```cpp
auto array = make_vectorized_array<VectorizedArray<Number, width>>(value);
```
indeed this case can be simply written (as used to be) as:
```cpp
auto array = make_vectorized_array<Number, width>(value);
```

However, I am tackling in this PR another case: the type `VectorizedArrayType` is given (instead of `Number` and `width`). If I would use the old `make_vectorized_array<Number, width>`,  I would have to write:
```cpp
auto array = make_vectorized_array<VectorizedArrayType::value_type, 
                                   VectorizedArrayType::n_array_elements>(value);
```
If I would use the new proposed function, I could simply write:
```cpp
auto array = make_vectorized_array<VectorizedArrayType>(value);
```
This makes the life of the user somewhat easier, since he/she does not have to think of the vector length explicitly.